core/controllers/base.py

```diff
@@ -175,6 +174,9 @@ def __init__(self, request, response):  # pylint: disable=super-init-not-called
         self.current_user_is_super_admin = False
         self.normalized_request = None
         self.normalized_payload = None
+        self.handler_args = {}
```

Comment:
Making these to a class level variable because -- Maybe in future we want to increase functionality like the SVS architecture so it will be great to have them class level constant.